builtin/mainmenu/dlg_csm.lua

```diff
@@ -0,0 +1,193 @@
+					conffile:set("load_mod_" .. mod.name,
+						mod.enabled and "true" or "false")
+				elseif mod.enabled then
+					gamedata.errormessage = fgettext_ne("Failed to enable clientmod" ..
+						" \"$1\" as it contains disallowed characters. " ..
+						"Only characters [a-z0-9_] are allowed.",
+						mod.name)
+				end
+				mods["load_mod_" .. mod.name] = nil
+			end
+		end
+
+		-- Remove mods that are not present anymore
+		for key in pairs(mods) do
+			if key:sub(1, 9) == "load_mod_" then
+				conffile:remove(key)
+			end
+		end
+		if not conffile:write() then
+			core.log("error", "Failed to write clientmod config file")
+		end
+	end
+
+	if tab_selected_pkg == nil then
+		tab_selected_pkg = 1
+	end
+
+	local use_technical_names = core.settings:get_bool("show_technical_names")
+
+
+	local fs = {
+		"size[6.5,7]",
+		"label[0.25,-0.1;", fgettext("Installed Client Mods"), "]",
+		"tablecolumns[color;tree;text]",
+		"table[0.15,0.5;6,5.7;pkglist;", pkgmgr.render_packagelist(packages, use_technical_names), ";", tab_selected_pkg, "]",
+		"button[0.15,5.5;3,3;back;", fgettext("Back"), "]"
+	}
+
+
+	local selected_pkg
+	if filterlist.size(packages) >= tab_selected_pkg then
+		selected_pkg = packages:get_list()[tab_selected_pkg]
+	end
+
+	if selected_pkg ~= nil then
+		if selected_pkg.type == "mod" then
+			if selected_pkg.is_clientside then
+				if selected_pkg.enabled then
+					table.insert_all(fs, {
+						"button[3.36,5.5;3,3;btn_csm_mgr_disable_mod;", fgettext("Disable"), "]"
+					})
+				else
+					table.insert_all(fs, {
+						"button[3.36,5.5;3,3;btn_csm_mgr_enable_mod;", fgettext("Enable"), "]"
+					})
+				end
+			end
+		end
+	end
+	return table.concat(fs, "")
+end
+
+--------------------------------------------------------------------------------
+local function handle_doubleclick(pkg, pkg_name)
+	pkgmgr.enable_mod({data = {list = packages, selected_mod = pkg_name}})
+	packages = nil
+end
+
+--------------------------------------------------------------------------------
+local function handle_buttons(dlgview, fields, tabname, tabdata)
+	if fields["pkglist"] ~= nil then
+		local event = core.explode_table_event(fields["pkglist"])
+		tab_selected_pkg = event.row
+		if event.type == "DCL" then
+			handle_doubleclick(packages:get_list()[tab_selected_pkg], tab_selected_pkg)
+		end
+		return true
+	end
+
+	if fields.btn_csm_mgr_mp_enable ~= nil or fields.btn_csm_mgr_mp_disable ~= nil then
+		pkgmgr.enable_mod({data = {list = packages, selected_mod = tab_selected_pkg}}, fields.btn_csm_mgr_mp_enable ~= nil)
+		packages = nil
+		return true
+	end
+
+	if fields.btn_csm_mgr_enable_mod ~= nil or fields.btn_csm_mgr_disable_mod ~= nil then
+		pkgmgr.enable_mod({data = {list = packages, selected_mod = tab_selected_pkg}}, fields.btn_csm_mgr_enable_mod ~= nil)
+		packages = nil
+		return true
+	end
+
+	if fields.back then
+		dlgview:delete()
+		return true
+	end
+
+	if fields["btn_csm_mgr_rename_modpack"] ~= nil then
```

Comment:
```suggestion
	if fields.btn_csm_mgr_rename_modpack then
```
The same for the other occurrences. It makes the code look more consistent. Currently you're mixing two different styles.